tls/s2n_tls13.c

```diff
@@ -16,6 +16,12 @@
 #include "api/s2n.h"
 #include "tls/s2n_tls.h"
 #include "tls/s2n_tls13.h"
+#include "crypto/s2n_rsa_signing.h"
+
+int s2n_is_tls13_supported()
+{
+    return s2n_is_rsa_pss_signing_supported();
```

Comment:
we should probably revisit the requirements of enabling tls13, but this should suffice for now